docs/UserGuide.md

```diff
@@ -116,17 +116,17 @@ This detailed guide will walk you through how to make the most of **EduManage**
 
 | Index |     Parameter     |                     Format                     | Constraints                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 |:-----:|:-----------------:|:----------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-|   1   |      Address      |                  `a/ADDRESS`                   | Can take any values except prefixes already reserved, and should not be blank.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+|   1   |      Address      |                  `a/ADDRESS`                   | Can take any values, but should not be blank. Note that using certain prefixes may cause an error.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Change to "Note that using reserved prefixes within the address may cause an error." for clarity?